samples/Microsoft.TestPlatform.TranslationLayer.E2ETest/Program.cs

```diff
@@ -51,6 +51,8 @@ public static int Main(string[] args)
                 }
             }
 
+            Console.WriteLine(Directory.GetCurrentDirectory());
```

Comment:
Should we prefix this output with `Current directory: {0}` to give context when the tool is run?